Evolutionary_loop/Archived/sspermissioncheck.py

```diff
@@ -0,0 +1,82 @@
+#*******************************************************************************
+#  file: sspermissioncheck.py
+#     Based on ssPermissionCheck.sh, this is an attempt to migrate to Python.
+#     This is currently left inside Archived as a python template
+#     but will be deleted eventually, as we no longer use ssPermissionCheck.sh
+#
+#  Programmer: Jason Yao (yao.966@osu.edu)
+#
+#  Revision history:
+#     07/18/23  Original version
+#
+#  Notes:
+#     * vertical ruler at column 80
+#
+#  TODO: 
+#     * ssPermissionCheck.sh sends email with error messages
+#
+#*******************************************************************************
+'''
+This script checks if the user-specified checkpoint (usr_spec) matches what
+is actually stored inside the SaveState.txt files.
+
+Usage: python3 sspermissioncheck.py <run_name> <working_dir> <usr_spec>
+
+For more information on the arguments: 
+  >> python3 sspermissioncheck.py -h
+'''
+
+from pathlib import Path
+import argparse
+import time
+
+
+def main(file_path:str):
```

Comment:
This is actually a `Path` object that you are passing, so the type hint is misleading.  You could fix it or remove it altogether, since it's already clear from the name that it's a path.  FTR Python doesn't enforces type automatically, and they are there more as documentation for the reader and for third-party tools (hence the name, type *hints*).

Reply:
Got it, removed!
Just curious, if we did not `import pathlib`, would the argument of `main(file_path)` still be a Path object somehow, or would it just be a `str` in that case?